MIGRATION-GUIDE.md

```diff
@@ -0,0 +1,85 @@
+# Migrating to JavaScript Browser SDK v1
+
+JavaScript Browser SDK v1.0.0 has a few breaking changes that you should consider when migrating from version 0.x.x.
+
+## Renamed some TypeScript definitions in the `SplitIO` namespace to avoid conflicts with other Split packages
+
+The renamed types are:
+- `SplitIO.IBrowserSettings` -> `SplitIO.IClientSideSettings`
+- `SplitIO.IBrowserAsyncSettings` -> `SplitIO.IClientSideAsyncSettings`
+- `SplitIO.ISDK` -> `SplitIO.IBrowserSDK`
+- `SplitIO.IAsyncSDK` -> `SplitIO.IBrowserAsyncSDK`
+- `SplitIO.IClient` -> `SplitIO.IBrowserClient`
+- `SplitIO.IAsyncClient` -> `SplitIO.IBrowserAsyncClient`
+
+For example, you should replace:
+
+```ts
+import { SplitFactory } from '@splitsoftware/splitio-browserjs';
+
+const config: SplitIO.IBrowserSettings = { ... };
+const factory: SplitIO.ISDK = SplitFactory(config);
+```
+
+with:
+
+```ts
+import { SplitFactory } from '@splitsoftware/splitio-browserjs';
+
+const config: SplitIO.IClientSideSettings = { ... };
+const factory: SplitIO.IBrowserSDK = SplitFactory(config);
+```
+
+
+## Removed the `LocalhostFromObject` export from the default import
+
+In order to simplify the SDK API, the `LocalhostFromObject` export was removed from the default import (`import { LocalhostFromObject } from '@splitsoftware/splitio-browserjs'`), and it is no longer necessary to manually pass it to the `sync.localhostMode` configuration option to enable localhost mode.
+
+If you were using the `LocalhostFromObject` export, simple remove it from your code. For example, replace:
```

Comment:
```suggestion
If you were using the `LocalhostFromObject` export, you should remove it from your code. For example, replace:
```
'simply' can be used here too (though not preferred), but just be careful about spelling